app/src/main/java/com/zeapo/pwdstore/crypto/PasswordCreationActivity.kt

```diff
@@ -430,19 +426,13 @@ class PasswordCreationActivity : BasePgpActivity(), OpenPgpServiceConnection.OnB
                                     returnIntent.putExtra(RETURN_EXTRA_USERNAME, username)
                                 }
 
-                                val repo = PasswordRepository.getRepository(null)
-                                if (repo != null) {
-                                    val status = Git(repo).status().call()
-                                    if (status.modified.isNotEmpty()) {
-                                        lifecycleScope.launch {
-                                            commitChange(
-                                                getString(
-                                                    R.string.git_commit_edit_text,
-                                                    getLongName(fullPath, repoPath, editName)
-                                                )
-                                            )
-                                        }
-                                    }
+                                lifecycleScope.launch {
```

Comment:
Modification checks have been rolled into `commitChange` so this too is unnecessary, and was also blocking.